File: touralbum/app/src/main/java/com/example/touralbum/ui/reminder/MemoApplication.java
package com.example.touralbum.ui.reminder;

import android.app.Application;
import android.content.Context;

public class MemoApplication extends Application{
    public static Context mContext;

    @Override
    public void onCreate() {
        super.onCreate();
        mContext = this;
    }

    public static Context getContext() {
        return mContext;
    }
}

File: touralbum/app/src/main/java/com/example/touralbum/ui/reminder/MemoApplication.kt
package com.example.touralbum.ui.reminder

import android.app.Application
import android.content.Context

class MemoApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        context = this
    }

    companion object {
        var context: Context? = null
            private set
    }
}